server/persistence/mysql/errors.go

```diff
@@ -0,0 +1,25 @@
+package mysql
+
+import (
+	"errors"
+
+	"github.com/go-sql-driver/mysql"
+)
+
+var (
+	errDuplicateEntry = errors.New(`mysql duplicate entry`)
+)
+
+func convertMysqlError(err error) error {
+	if err == nil {
+		return nil
+	}
+	switch t := err.(type) {
+	case *mysql.MySQLError:
+		switch t.Number {
+		case 1062:
```

Comment:
:cactus: can we make the magic number a const?

Reply:
I looked in the golang mysql driver and couldn't find a const mapping there. I think this is gonna hafta do. It's pulled out of the mysql spec here: https://dev.mysql.com/doc/refman/5.7/en/server-error-reference.html#error_er_dup_entry

Reply:
Maybe a comment then so it's not so 🧙